### PR TITLE
evemu: New package

### LINKED ISSUE #26600: package request: evemu
A tool for debugging kernel input devices.

[Freedesktop page](https://www.freedesktop.org/wiki/Evemu/)

### PR DESCRIPTION
**Pull request by ndowens**
Closes #26600

### CODE REVIEW

File: srcpkgs/evemu/template [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+version=2.7.0
+revision=1
+build_style=gnu-configure
+hostmakedepends="automake pkg-config libtool python3"
+makedepends="libevdev-devel"
+short_desc="Kernel device emulation"
+maintainer="Nathan Owens <ndowens@artixlinux.org>"
+license="LGPL-3.0-only"
+homepage="https://github.com/freedesktop/evemu"
+distfiles="https://github.com/freedesktop/evemu/archive/v${version}.tar.gz"
```
Comment (sgn) [inline srcpkgs/evemu/template:12]:
```
homepage="https://freedesktop.org/wiki/Evemu/"
distfiles=https://www.freedesktop.org/software/evemu/evemu-$version.tar.xz
```

**Comment by sgn** [comment]
```diff
diff --git a/srcpkgs/evemu/template b/srcpkgs/evemu/template
index 12513aae59..2cbbaeb9c6 100644
--- a/srcpkgs/evemu/template
+++ b/srcpkgs/evemu/template
@@ -3,7 +3,7 @@ pkgname=evemu
 version=2.7.0
 revision=1
 build_style=gnu-configure
-hostmakedepends="automake pkg-config libtool python3"
+hostmakedepends="pkg-config python3 asciidoc xmlto"
 makedepends="libevdev-devel"
 short_desc="Kernel device emulation"
 maintainer="Nathan Owens <ndowens@artixlinux.org>"
@@ -11,7 +11,3 @@ license="LGPL-3.0-only"
 homepage="https://freedesktop.org/wiki/Evemu/"
 distfiles=${FREEDESKTOP_SITE}/evemu/evemu-$version.tar.xz
 checksum=78c9400d55eeeb5ab75161360543f9376438c4da4934cb34cdda5b46021ae379
-
-pre_configure() {
-	autoreconf -fi
-}
```

I don't know if splitting `-devel` is a good idea.

**Comment by ndowens** [comment]
> ```diff
> diff --git a/srcpkgs/evemu/template b/srcpkgs/evemu/template
> index 12513aae59..2cbbaeb9c6 100644
> --- a/srcpkgs/evemu/template
> +++ b/srcpkgs/evemu/template
> @@ -3,7 +3,7 @@ pkgname=evemu
>  version=2.7.0
>  revision=1
>  build_style=gnu-configure
> -hostmakedepends="automake pkg-config libtool python3"
> +hostmakedepends="pkg-config python3 asciidoc xmlto"
>  makedepends="libevdev-devel"
>  short_desc="Kernel device emulation"
>  maintainer="Nathan Owens <ndowens@artixlinux.org>"
> @@ -11,7 +11,3 @@ license="LGPL-3.0-only"
>  homepage="https://freedesktop.org/wiki/Evemu/"
>  distfiles=${FREEDESKTOP_SITE}/evemu/evemu-$version.tar.xz
>  checksum=78c9400d55eeeb5ab75161360543f9376438c4da4934cb34cdda5b46021ae379
> -
> -pre_configure() {
> -	autoreconf -fi
> -}
> ```
> 
> I don't know if splitting `-devel` is a good idea.

Splitting what? At first I thought meant I made a subpackage, e.g evemu-devel but only -devel I see is evdev-devel?

**Comment by sgn** [comment]
> > 
> > I don't know if splitting `-devel` is a good idea.
> 
> Splitting what? At first I thought meant I made a subpackage, e.g evemu-devel but only -devel I see is evdev-devel?

`/usr/include/` and `/usr/lib/*.a`

**Comment by sgn** [comment]
> ```diff
> diff --git a/srcpkgs/evemu/template b/srcpkgs/evemu/template
> index 12513aae59..2cbbaeb9c6 100644
> --- a/srcpkgs/evemu/template
> +++ b/srcpkgs/evemu/template
> @@ -3,7 +3,7 @@ pkgname=evemu
>  version=2.7.0
>  revision=1
>  build_style=gnu-configure
> -hostmakedepends="automake pkg-config libtool python3"
> +hostmakedepends="pkg-config python3 asciidoc xmlto"
>  makedepends="libevdev-devel"
>  short_desc="Kernel device emulation"
>  maintainer="Nathan Owens <ndowens@artixlinux.org>"
> @@ -11,7 +11,3 @@ license="LGPL-3.0-only"
>  homepage="https://freedesktop.org/wiki/Evemu/"
>  distfiles=${FREEDESKTOP_SITE}/evemu/evemu-$version.tar.xz
>  checksum=78c9400d55eeeb5ab75161360543f9376438c4da4934cb34cdda5b46021ae379
> -
> -pre_configure() {
> -	autoreconf -fi
> -}
> ```
> 
> I don't know if splitting `-devel` is a good idea.

I think I was pretty clear on applying this patch?
And please check license carefully, next time.

**Pull request closed without merging**